dandiapi/api/mail.py

```diff
@@ -1,14 +1,21 @@
 import logging
-from typing import List
+from typing import Any, Dict, List
 
 from allauth.socialaccount.models import SocialAccount
+from django.conf import settings
 from django.contrib.auth.models import User
 from django.core import mail
-from django.template.loader import render_to_string
+from django.template.loader import render_to_string as _render_to_string
 
 logger = logging.getLogger(__name__)
 
 
+def render_to_string(template: str, render_context: Dict[str, Any]):
+    render_context['dandi_api_url'] = settings.DANDI_API_URL
+    render_context['dandi_web_app_url'] = settings.DANDI_WEB_APP_URL
+    return _render_to_string(template, render_context)
```

Comment:
```suggestion
from django.template.loader import render_to_string

logger = logging.getLogger(__name__)


BASE_RENDER_CONTEXT = {
    'dandi_api_url': settings.DANDI_API_URL,
    'dandi_web_app_url': settings.DANDI_WEB_APP_URL,
}
```

Then you can just do something like this whenever you need to customize the render context:
```
render_context = {
    **BASE_RENDER_CONTEXT,
    'foo': 'bar',
}
```
That reads easier to me than doing function composition, but the difference is purely cosmetic, so your call.